### PR TITLE
fix!: Login services button colors

### PR DESCRIPTION
**Pull request by yash-rajpal**
## Proposed changes (including videos or screenshots)

The login services button was not implementing the button color properties, so fixed it and now the login services button will render the button colors as per the settings.

Also added the alerts for all button-color settings.

Implemented a migration for existing workspaces, to update all the button and label colors setting values according to our design system.

image

image

## Further comments

[SUP-590](https://rocketchat.atlassian.net/browse/SUP-590)

[SUP-590]: https://rocketchat.atlassian.net/browse/SUP-590?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .changeset/fifty-mails-admire.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+'@rocket.chat/web-ui-registration': patch
+"@rocket.chat/meteor": minor
```
Comment (MarcosSpessatto) [inline .changeset/fifty-mails-admire.md:3]:
```suggestion
"@rocket.chat/meteor": major
```

File: apps/meteor/server/startup/migrations/v313.ts [hunk 2]
```diff
@@ -0,0 +1,118 @@
+import type { ILoginServiceConfiguration, OAuthConfiguration } from '@rocket.chat/core-typings';
+import { Settings, LoginServiceConfiguration } from '@rocket.chat/models';
+
+import { settings } from '../../../app/settings/server';
```
Comment (MarcosSpessatto) [inline apps/meteor/server/startup/migrations/v313.ts:4]: We can't use the cached setting in migrations, please use the model instead.

File: apps/meteor/tests/e2e/utils/convertHexToRGB.ts
Comment (MarcosSpessatto) [inline apps/meteor/tests/e2e/utils/convertHexToRGB.ts:1]: What about creating a unit-test for this function?

**Review by MarcosSpessatto — CHANGES_REQUESTED** [review]
Also, please write a description for the PR

File: apps/meteor/tests/e2e/utils/convertHexToRGB.ts
Comment (yash-rajpal) [reply apps/meteor/tests/e2e/utils/convertHexToRGB.ts:1]: This is a helper function for e2e tests, it is not being used anywhere in the main project, do we really need to test it?

File: apps/meteor/server/startup/migrations/v313.ts (hunk 2 above)
Comment (yash-rajpal) [reply apps/meteor/server/startup/migrations/v313.ts]: Done!

File: .changeset/fifty-mails-admire.md (hunk 1 above)
Comment (yash-rajpal) [reply .changeset/fifty-mails-admire.md]: Done!

File: apps/meteor/tests/e2e/utils/convertHexToRGB.ts
Comment (MarcosSpessatto) [reply apps/meteor/tests/e2e/utils/convertHexToRGB.ts:1]: Oh didn't see it 😛 nevermind then

**Review by MarcosSpessatto — APPROVED** [review]

**Comment by ggazzo** [comment]
merging before conflicts again

**Pull request merged**